<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Recursos/alta-temperatura.png</normaloff>:/Recursos/alta-temperatura.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
border-left-color: rgb(85, 170, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>Centigrados</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLCDNumber" name="outCent">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 0, 255);
</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="inCent">
         <property name="cursor">
          <cursorShape>ClosedHandCursor</cursorShape>
         </property>
         <property name="minimum">
          <number>-18</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>Fahrenheit</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLCDNumber" name="outFahr">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 170, 255);</string>
         </property>
         <property name="value" stdset="0">
          <double>32.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="inFahr">
         <property name="cursor">
          <cursorShape>ClosedHandCursor</cursorShape>
         </property>
         <property name="maximum">
          <number>210</number>
         </property>
         <property name="value">
          <number>32</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);</string>
      </property>
      <property name="title">
       <string>Kelvin</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QDial" name="inKel">
         <property name="cursor">
          <cursorShape>ClosedHandCursor</cursorShape>
         </property>
         <property name="maximum">
          <number>373</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLCDNumber" name="outKel">
         <property name="font">
          <font>
           <pointsize>6</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 85, 255);</string>
         </property>
         <property name="smallDecimalPoint">
          <bool>true</bool>
         </property>
         <property name="digitCount">
          <number>5</number>
         </property>
         <property name="value" stdset="0">
          <double>273.149999999999977</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1130</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>inCent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outCent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inFahr</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outFahr</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>643</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inKel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outKel</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>662</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>667</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
